hub/core/storage/provider.py

```diff
@@ -125,3 +127,13 @@ def flush(self):
         """Only needs to be implemented for caches. Flushes the data to the next storage provider.
         Should be a no op for Base Storage Providers like local, s3, azure, gcs, etc.
         """
+
+    def enable_autoflush(self):
+        self.autoflush = True
+
+    def disable_autoflush(self):
+        self.autoflush = False
```

Comment:
maybe `storage.autoflush = True` can be set instead? and have `@autoflush.setter` to make sure it's a bool?

Reply:
lol setters and getters pyjava

Reply:
I like that, but I won't even include the bool check. I'll just swap these function calls for a direct write